examples/neutra.py

```diff
@@ -64,26 +67,19 @@ def main(args):
     svi_init, svi_update, _ = svi(dual_moon_model, guide, elbo, opt_init, opt_update, get_params)
     opt_state, _ = svi_init(rng_init)
 
-    def body_fn(val):
-        i, loss, opt_state_, rng_ = val
+    def body_fn(val, i):
```

Comment:
nit: can we keep the order (i, val) instead to match the ordering in `lax.fori_loop`'s `body_fn`?

Reply:
Here I want to use `lax.scan`, so i is just a dummy variable (though it is currently used in `svi_update`).